### PR TITLE
Add description for some images

### PR DESCRIPTION
**Pull request by serg**

### CODE REVIEW

**Review by embik — COMMENTED** [review]
Please mirror the changes to `content/kubermatic/main` as well.

**Comment by serg** [comment]
@embik nice point, thanks! 👍
Updated

File: content/kubermatic/main/tutorials-howtos/administration/admin-panel/_index.en.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ $ kubectl edit user ...
 After logging in to the dashboard as an administrator, you should be able to access the admin panel from the menu up
 top.
 
-![](@/images/ui/admin-panel-access.png?height=300px&classes=shadow,border "Accessing the Admin Panel")
+![Accessing the admin panel of Kubermatic Kubernetes Platform](/img/kubermatic/v2.25/ui/admin-panel-access.png?height=300px&classes=shadow,border "Accessing the Admin Panel")
 
-![](@/images/ui/panel.png?height=350px&classes=shadow,border "Admin Panel")
+![Admin panel interface of Kubermatic Kubernetes Platform](/img/kubermatic/v2.25/ui/panel.png?classes=shadow,border "Admin Panel")
```
Comment (embik) [inline content/kubermatic/main/tutorials-howtos/administration/admin-panel/_index.en.md:21]: Please don't override the `@` short code, it points to the images for this version so we no longer need the `/v2.25/` / `/main` part. This applies to all the pages in `content/kubermatic/main`, we don't have that in older version of the docs.

**Review by embik — CHANGES_REQUESTED** [review]

File: content/kubermatic/main/tutorials-howtos/administration/admin-panel/_index.en.md (hunk 1 above)
Comment (serg) [reply content/kubermatic/main/tutorials-howtos/administration/admin-panel/_index.en.md:21]: sorry, missed this, I will update

**Review by embik — APPROVED** [review]
/approve

**Pull request merged**